pixart.js

```diff
@@ -1 +1,14 @@
+$("#set-color").on("click", function() {
+  event.preventDefault();
+  var color = $("#color-field").val();
+  $(".brush").css("background-color", color);
+});
+var setBrushColor = $(".button").on("click", setBrushColor);
```

Comment:
I suspect this line will throw an error... because there is no `setBrushColor` function that you've defined. But the good news is that your side above (lines 1 - 5) do what you probably intended to do here.